src/GovUk.Education.ExploreEducationStatistics.Admin/Services/UserManagementService.cs

```diff
@@ -166,69 +174,77 @@ public async Task<UserViewModel> GetAsync(string userId)
             return null;
         }
 
-        public async Task<List<UserViewModel>> ListPendingAsync()
+        public async Task<Either<ActionResult, List<UserViewModel>>> ListPendingInvites()
         {
-            var pendingUsers = await _usersAndRolesDbContext.UserInvites.Where(u => u.Accepted == false)
-                .OrderBy(x => x.Email).Select(u => new UserViewModel
-                {
-                    Email = u.Email,
-                    Role = u.Role.Name
-                }).ToListAsync();
-
-            return pendingUsers;
+            return await _userService
+                .CheckCanManageAllUsers()
+                .OnSuccess(async () =>
+                    await _usersAndRolesDbContext.UserInvites
+                        .Where(ui => ui.Accepted == false)
+                        .OrderBy(ui => ui.Email)
+                        .Include(ui => ui.Role)
```

Comment:
Just so you know, adding this `Include` last hasn't altered the order of execution here in case you did this thinking that it's an optimisation. It's just another thing added into what gets translated into the resulting SQL query.